pkg/signatures/cosign_sig_fetcher.go

```diff
@@ -131,6 +146,30 @@ func (c *cosignSignatureFetcher) FetchSignatures(ctx context.Context, image *sto
 	return cosignSignatures, nil
 }
 
+func certificateFromSignedPayload(sp cosign.SignedPayload) ([]byte, error) {
+	if sp.Cert == nil {
+		return nil, nil
```

Comment:
That's counter intuitive on the other hand it's a slice so it might works
How about returning an empty slice?

Reply:
Since we store this within the `CertPem/CertChainPem` fields wanted to avoid storing an empty slice (i.e. when declaring `[]byte{}`), and just doing 
```go
var slice []byte
if sp.Cert == nil {
    return slice, nil
}
``` would be the same than just returning nil 🤷